rmgpy/cantherm/thermo.py

```diff
@@ -179,7 +179,16 @@ def save(self, outputFile):
         string = writeThermoEntry(species, elementCounts=elementCounts, verbose=False)
         f.write('{0}\n'.format(string))
         f.close()
-    
+
+        # write species dictionary
+        f = open(os.path.join(os.path.dirname(outputFile), 'species_dictionary.txt'), 'a')
+        if isinstance(species, Species):
+            if species.molecule and isinstance(species.molecule[0], Molecule):
+                f.write(species.label)
+                f.write('\n')
+                f.write(species.molecule[0].toAdjacencyList(removeH=False))
```

Comment:
You can pass a `label` argument to `toAdjacencyList` instead of writing the label above.

Reply:
sounds like a much better strategy. I'll fix it.